README.min.md

```diff
@@ -16,17 +16,17 @@ cd local_deploy
 python deploy_ds.py  # Tested with Python 3.7.13 
 ```
 
-This will start the minimal SQUARE service and run the example script `local_deploy/deploy_ds.py` to 
-- Download example corpus scifact;
-- Upload the corpus (to the ES in SQUARE);
-- Do BM25 search;
-- Embed and index the corpus with DPR (GPU 1min, CPU 30min);
-- Start a Faiss container to load the built dense index;
-- Do dense search.
+> This will start the minimal SQUARE service and run the example script `local_deploy/deploy_ds.py` to 
+> - Download example corpus scifact;
+> - Upload the corpus (to the ES in SQUARE);
+> - Do BM25 search;
+> - Embed and index the corpus with DPR (GPU 1min, CPU 30min);
+> - Start a Faiss container to load the built dense index;
+> - Do dense search.
 
-One can also get access to the [SWAGGER UI](https://swagger.io/tools/swagger-ui/) page for the interactive API. To do so, one can do 
+One can also get access to the [SWAGGER UI](https://swagger.io/tools/swagger-ui/) page for the interactive API of Datastores. To do so, one can do 
 ```bash
 cd local_deploy
-python get_token.py
+python get_token.py  # Or run `square_token` from command line
```

Comment:
why do we still need this get_token.py?

Reply:
OK. I have deleted it and told people to use square_token instead [07bfdc6](https://github.com/UKP-SQuARE/square-core/pull/354/commits/07bfdc6245b25ea9dfc1f495e48432532d2cf660)